### PR TITLE
refactor: replace BestLogger with oslog

### PR DESCRIPTION
**Pull request by michaellee8**

### CODE REVIEW

File: Sources/TorusSwiftDirectSDK/TorusSwiftDirectSDKFactory.swift [hunk 1]
```diff
@@ -21,7 +23,23 @@ public class TDSDKFactory: TDSDKFactoryProtocol{
         return FetchNodeDetails(proxyAddress: net, network: network)
     }
     
-    public func createTorusUtils(level: BestLogger.Level, nodePubKeys: Array<TorusNodePub> = []) -> AbstractTorusUtils {
-        return TorusUtils(label: "TorusUtils", loglevel: level, nodePubKeys: nodePubKeys)
+    public func createTorusUtils(level: OSLogType, nodePubKeys: Array<TorusNodePub> = []) -> AbstractTorusUtils {
```
Comment (rathishubham7) [inline Sources/TorusSwiftDirectSDK/TorusSwiftDirectSDKFactory.swift:25]: Why is this required ?

Comment (michaellee8) [reply Sources/TorusSwiftDirectSDK/TorusSwiftDirectSDKFactory.swift:25]: you mean i should just remove the level argument right now?

Comment (rathishubham7) [reply Sources/TorusSwiftDirectSDK/TorusSwiftDirectSDKFactory.swift:25]: Why are you using BestLogger.Level here ? It is not needed.

Comment (michaellee8) [reply Sources/TorusSwiftDirectSDK/TorusSwiftDirectSDKFactory.swift:25]: Oh sorry I only saw the diffs above the comment here. Yes it shouldn't be here, let me change it.

Comment (michaellee8) [reply Sources/TorusSwiftDirectSDK/TorusSwiftDirectSDKFactory.swift:25]: @rathishubham7 Is the latest version good to go?
